tokens/eth/0x2C4e8f2D746113d0696cE89B35F0d8bF88E0AEcA.json

```diff
@@ -6,28 +6,28 @@
   "ens_address": "",
   "website": "https://simpletoken.org",
```

Comment:
shouldn't this then also be ost.com?

Reply:
That's a good question, thank you for asking! That URL forwards to ost.com, which is our current preference for managing this shift.

Reply:
OK merging it then if it is your preference - just feels a bit inconsistent to me